debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/sink/MongoDbSinkConnectorTask.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mongodb.sink;
+
+import java.util.Collection;
+import java.util.Map;
+
+import org.apache.kafka.clients.consumer.OffsetAndMetadata;
+import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.connect.errors.ConnectException;
+import org.apache.kafka.connect.sink.ErrantRecordReporter;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.apache.kafka.connect.sink.SinkTask;
+import org.apache.kafka.connect.sink.SinkTaskContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.mongodb.client.MongoClient;
+import com.mongodb.internal.VisibleForTesting;
+
+import io.debezium.config.Configuration;
+import io.debezium.connector.mongodb.connection.MongoDbConnectionContext;
+import io.debezium.dlq.ErrorReporter;
+
+public class MongoDbSinkConnectorTask extends SinkTask {
+    static final Logger LOGGER = LoggerFactory.getLogger(MongoDbSinkConnectorTask.class);
```

Comment:
```suggestion
    private static final Logger LOGGER = LoggerFactory.getLogger(MongoDbSinkConnectorTask.class);
```

Reply:
No, this is actually needed as the sub-task `io.debezium.connector.mongodb.sink.StartedMongoDbSinkTask` aggregates its logs under this `MongoDbSinkConnectorTask` logger.

Reply:
@rk3rn3r In this case I'd prefere `protected`

Reply:
I think that won't work as there is not inheritance. That's why it's `package-private`, which is imo even stricter than protected, no?

Reply:
Well, your comment says `sub-class` that's why I expected the inheritance :-)